packages/grid/.storybook/main.js

```diff
@@ -0,0 +1,6 @@
+const storybookConfig = require( '@automattic/calypso-storybook' );
```

Comment:
I'm basically copying what we already do in multiple packages like the components one. But I do wonder whether a single storybook for the whole repo is better than these smaller ones.